app/assets/javascripts/base.js.erb

```diff
@@ -1,4 +1,6 @@
 $(function() {
+  var SEARCH_DELIMITER = '<%= SearchQuerySanitizationService::DELIMITER %>';
```

Comment:
We can no longer use a space character as the delimiter, since some fields we search by now contain spaces (e.g. topic tags).